modules/packages/Sort.chpl

```diff
@@ -1235,7 +1235,7 @@ module QuickSort {
   proc quickSortImpl(Data: [?Dom] ?eltType,
                      minlen=16,
                      comparator:?rec=defaultComparator,
-                     start:int = Dom.low, end:int = Dom.high) {
+                     start:int = Dom.lowBound, end:int = Dom.highBound) {
```

Comment:
I'm not sure, but I would think that all of these low/high in Sort should be alignedLow / alignedHigh, because sorting will generally use these indices to access into an array. But it also might be that this code does not run for strided arrays. I do not remember.

Reply:
I've changed these and added a variation of one of our "try a bunch of sorts" correctness tests to sort strided arrays, but strangely it worked with 1.26.0 as well, so either these routines are not sensitive to whether the bounds are aligned or not, or my test isn't triggering the code paths that would be sensitive to it for some reason.  In any case, I've made the change to use the aligned bounds.